portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,168 +49,203 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-
+ 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private int numberOfCommentsToDisplay = 0;
-  private List<String> tagsToSearch = new ArrayList<String>();
   private final static String MESSAGE_PARAMETER = "text-input";
   private final static String SENDER_PARAMETER = "sender";
   private final static String TAG_PARAMETER = "tags";
-  
-  // TODO: ADD TO FORM IN INDEX
-  private final static long PARENT_ID_PARAMETER = 0;
```

Comment:
I would ask not to delete this because I am going to update to actually get form input so then I'll have to go find where I placed the  PARENT_ID_PARAMETER

Reply:
My bad i should've asked. So is it like a parameter name like the ones above it? I saw that you assigned 0 to it so i made it a local variable instead. We can meet to resolve conflicts.